storage/volume/driver/proxy.go

```diff
@@ -0,0 +1,173 @@
+package driver
+
+import (
+	"errors"
+
+	"github.com/alibaba/pouch/plugins"
+)
+
+const (
+	remoteVolumeCreateService       = "/VolumeDriver.Create"
```

Comment:
What is the usage of all the const variables? Could you add a little bit annotation for these? 
Since it would be quite hard to understand this part of codes with no guidance.

Reply:
These const variables is the part of remote volume protocols, I will add more annotations